src/gpu_common.h

```diff
@@ -51,6 +51,12 @@ extern int cool_gpu_down;
 
 extern char gpu_degree_sign[8];
 
+/* Passive init: enumerate platforms and devices and parse options */
+void opencl_load_environment(void);
+
+/* Get the number of available devices (all the OpenCL devices) */
+int get_number_of_available_devices();
```

Comment:
Should also have `(void)` for consistency.

Also, is this header file included in the source file(s) that define those functions, to let the compiler check that the prototypes match the definitions and warn us if not? It should be, or the prototypes above should be in a header file that is.